channels-rke2.yaml

```diff
@@ -434,10 +419,10 @@ releases:
         repo: rancher-rke2-charts
         version: 0.1.1000
       rancher-vsphere-cpi:
-        repo: rancher-charts
+        repo: rancher-rke2-charts
         version: 1.1.000
       rancher-vsphere-csi:
-        repo: rancher-charts
+        repo: rancher-rke2-charts
```

Comment:
@phillipsj @rosskirkpat Are these the valid CPI/CSI versions for 1.22 from the rke2-charts?

Reply:
@deniseschannon yes those looks correct. They match the versions that they shipped with that release. https://github.com/rancher/rke2/blob/release-1.22/Dockerfile#L113-L114